dev/com.ibm.websphere.appserver.features/visibility/public/mpConcurrency-1.0/resources/l10n/com.ibm.websphere.appserver.mpConcurrency-1.0.properties

```diff
@@ -14,4 +14,4 @@
 #NLS_MESSAGEFORMAT_NONE
 #
 
-description=This feature enables JAX-RS async invocations to return instances of CompletableFuture that are backed by a ManagedExecutorService.
\ No newline at end of file
+description=This feature implements the MicroProfile Concurrency 1.0 specification, which allows you to obtain instances of CompletableFuture that are backed by a ManagedExecutor and provides the ability to contextualize CompletableFuture actions.
```

Comment:
My only comment is to consider changing ManagedExecutor to managed executor.

Reply:
Fixing this according to subsequent comments under pull #5380.